server/vcs/markdown/templates/policyCheckSuccessUnwrapped.tmpl

```diff
@@ -1,11 +1,6 @@
 ```diff
 {{.PolicyCheckOutput}}
 ```
-
-{{ if ne .ApplyCmd  "" }}* :arrow_forward: To **apply** this plan, comment:
-    * `{{.ApplyCmd}}`{{end}}
-{{ if ne .LockURL  "" }}* :put_litter_in_its_place: To **delete** this plan click [here]({{.LockURL}}){{end}}
-{{ if ne .RePlanCmd  "" }}* :repeat: To re-run policies **plan** this project again by commenting:
-    * `{{.RePlanCmd}}`{{end}}{{ if .HasDiverged }}
```

Comment:
does this mean we have no more use for the underlying `ApplyCmd` and `RePlanCmd` fields? would we be able to remove them?